src/stable/test/fuzz.ts

```diff
@@ -1,23 +1,34 @@
 import { spawn } from 'child_process';
 import { CuzzConfig } from 'cuzz';
-import { readFile } from 'fs/promises';
+import { readFileSync } from 'fs';
 import { join } from 'path';
 
 import { getCanisterNames } from '.';
 
 export async function runFuzzTests(): Promise<void> {
-    const cuzzConfig = await getCuzzConfig();
+    const cuzzConfig = getCuzzConfig();
+
+    if (cuzzConfig.skip === true) {
```

Comment:
We should probably actually skip the fuzz tests using the jest skip functionality so that it shows up in the terminal UI as an actual skip